spec/unit/cli_spec.rb

```diff
@@ -597,6 +598,15 @@
         with("foo", "bar")
       run_command(["config", "foo=bar"])
     end
+
+    it "shows the alternative synopsis when the help is called" do
+     require 'byebug'
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.